site/documentation/content/api/Metrics.md

```diff
@@ -65,8 +65,19 @@ Additional tags for protocol adapters are:
 | *tenant*    | *string*                                           | The identifier of the tenant that the metric is being reported for |
 | *ttd*       | `command`, `expired`, `none`                    | A status indicating the outcome of processing a TTD value contained in a message received from a device.<br>`command` indicates that a command for the device has been included in the response to the device's request for uploading the message.<br>`expired` indicates that a response without a command has been sent to the device.<br>`none` indicates that either no TTD value has been specified by the device or that the protocol adapter does not support it. |
 | *type*      | `telemetry`, `event`                             | The type of (downstream) message that the metric is being reported for. |
+
+Additional tags for *hono.connections.attempts*:
+
+| Name        | Value                                              | Description |
+| ----------- | -------------------------------------------------- | ----------- |
 | *outcome*   | `adapter-disabled`, `connection-duration-exceeded`,<br/>`data-volume-exceeded`, `registration-assertion-failure`,<br/>`succeeded`, `tenant-connections-exceeded`,<br/>`unauthorized`, `unavailable`, `unknown` | The outcome of a device's connection attempt.<br/>`adapter-connections-exceeded` indicates that the maximum number of connections that the adapter instance can handle are exceeded<br/>`adapter-disabled` indicates that the protocol adapter is not enabled for the device's tenant<br/>`connection-duration-exceeded` indicates that the overall amount of time that a tenant's devices may be connected to an adapter has exceeded<br/>`data-volume-exceeded` indicates that the overall amount of data that a tenant's device may transfer per time period has exceeded<br/>`registration-assertion-failure` indicates that the device is either unknown or disabled<br/>`succeeded` indicates a successfully established connection<br/>`tenant-connections-exceeded` indicates that the maximum number of devices that may be connected simultaneously for a tenant has been exceeded<br/>`unauthorized` indicates that the device failed to authenticate<br/>`unavailable` indicates that some of Hono's (required) services are not available<br/>`unknown` indicates an unknown reason. |
 
+Additional tags for *hono.downstream.sent*:
+
+| Name        | Value                                              | Description |
+| ----------- | -------------------------------------------------- | ----------- |
+| *outcome*   | *string* (AMQP 1.0 disposition) | Any of the AMQP 1.0 disposition states, as well as `Aborted`, in the case the connection/link was closed before the disposition could be read. | 
```

Comment:
I think this is better. However, it is unclear, which values are actually allowed here.
The AMQP 1.0 spec defines the delivery states as `received`, `accepted`, `rejected`, `released` and `modified`.
The code currently uses the simple class names, i.e. starting with a capital letter, plus the non-AMQP state `Aborted`.
For the other tags we only use all lowercase values so far so I would prefer to stick to that pattern and explicitly list the possible values here. WDYT?

Reply:
It would have been more efficient to do this right away, I will amend the PR … anything else?

Reply:
No, that should be it FMPOV. Sorry for the inconvenience ...